### PR TITLE
Add intro to fragments to creating acc doc

### PR DESCRIPTION
**Pull request by trisberg**
Which other branches should this be merged with (if any)?

Only for 1.2.0

### CODE REVIEW

File: application-accelerator/creating-accelerators/creating-accelerators.md [hunk 1]
```diff
@@ -80,6 +80,127 @@ An alternative to using the Tanzu CLI is to create a separate manifest file and
...
+    ref:
+      branch: tap-1.2
+    url: https://github.com/sample-accelerators/fragments.git
+    subPath: java-version
+```
+
+To create the fragment (we can save the above manifest in a `java-version.yaml` file) and use:
+
+```
+tanzu accelerator apply -f ./java-version.yaml
```
Comment (gm2552) [inline application-accelerator/creating-accelerators/creating-accelerators.md:166]: You use `tanzu acc apply` to create fragments?  This initially confused me, so if this is true, you might want to clearly call out that the above command can be used for fragments and accelerators.

Comment (gm2552) [inline application-accelerator/creating-accelerators/creating-accelerators.md:166]: I'm seeing a mixture of kubectl and tanzu commands when applying manifest files in this document (see lines 80 and 166).  Is there a litmus test for when to use each one?

Comment (trisberg) [reply application-accelerator/creating-accelerators/creating-accelerators.md:166]: they basically do the same thing, the accelerator apply is a bit more picky - has to be a single Accelerator or Fragment resource in the yaml

Comment (trisberg) [reply application-accelerator/creating-accelerators/creating-accelerators.md:166]: I pushed another commit with a note to clarify

Comment (trisberg) [reply application-accelerator/creating-accelerators/creating-accelerators.md:166]: updated to use the `accelerator apply` command consistently in this doc

**Pull request merged**